tests/utils/utils.go

```diff
@@ -318,6 +352,86 @@ func SendERC20Destination(
 	return receipt, event.Amount
 }
 
+// Send a native token from fromBridge to toBridge via multihop through the C-Chain
+// Requires that both fromBridge and toBridge are fully collateralized
+// Requires that both fromBridge and toBridge have the same tokenMultiplier and multiplyOnReceive
+// with respect to the original asset on the C-Chain
+func SendNativeMultihopAndVerify(
+	ctx context.Context,
+	network interfaces.Network,
+	fundedKey *ecdsa.PrivateKey,
+	recipientKey *ecdsa.PrivateKey,
+	fromSubnet interfaces.SubnetTestInfo,
+	fromBridge *nativetokendestination.NativeTokenDestination,
+	toSubnet interfaces.SubnetTestInfo,
+	toBridge *nativetokendestination.NativeTokenDestination,
+	toBridgeAddress common.Address,
+	cChainInfo interfaces.SubnetTestInfo,
+	bridgedAmount *big.Int,
+	tokenMultiplier *big.Int,
+	multiplyOnReceive bool,
+) {
+	recipientAddress := crypto.PubkeyToAddress(recipientKey.PublicKey)
+	teleporterUtils.SendNativeTransfer(
+		ctx,
+		fromSubnet,
+		fundedKey,
+		recipientAddress,
+		big.NewInt(1e18),
+	)
+	input := nativetokendestination.SendTokensInput{
+		DestinationBlockchainID:  toSubnet.BlockchainID,
+		DestinationBridgeAddress: toBridgeAddress,
+		Recipient:                recipientAddress,
+		PrimaryFee:               big.NewInt(0),
+		SecondaryFee:             big.NewInt(0),
+		RequiredGasLimit:         DefaultNativeTokenRequiredGasLimit,
+	}
+	// Find the amount sent by fromBridge. This is before any scaling/unscaling is applied.
+	bridgedAmount = big.NewInt(0).Sub(bridgedAmount, input.PrimaryFee)
+
+	// Send tokens through a multihop transfer
+	originReceipt, _ := SendNativeTokenDestination(
+		ctx,
+		fromSubnet,
+		fromBridge,
+		input,
+		bridgedAmount,
+		recipientKey,
```

Comment:
Sorry, should have been more clear with my comment above. The `recipientKey` is being passed in here as `senderKey`. It seems strange, is it possibly misnamed?

Reply:
Gotcha. I adapted `SendERC20MultihopAndVerify` and kept some parameters that weren't strictly necessary. I renamed the parameters to that function to reflect that there are two separate keys involved - one sending ERC20s, and another providing gas to the first key. `SendNativeMultihopAndVerify` only needs one key, since all asset transfers are in the native token.